Pidaq/Lib/uploadToDrive/uploadToDrive.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function
+import os
+import logging
+from datetime import datetime
+
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+
+
+SCOPES = 'https://www.googleapis.com/auth/drive.file'
+store = file.Storage('credentials.json') 
+creds = store.get()
+
+if not creds or creds.invalid:
+    flow = client.flow_from_clientsecrets('client_secret.json', SCOPES)
+    creds = tools.run_flow(flow, store)
+    
+DRIVE = build('drive', 'v3', http=creds.authorize(Http()))
+
+
+
+def uploadCsv(filePath, parent):
+    """
+    Uploads a csv file from a local directory to a folder on Google Drive.
+
+    Parameters:
+    "filePath" is the path of the csv file itself and
+                should be passed without quotations.
+    "parent" is the name of the testing folder in the drive
+            and only a specific set of values are allowed.
+    The parent folder is supposedly going to be fixed for a
+    long period of time.
+    """
+    parentDictionary = {'Safety':'1Vnb9UOTatdZUpcBXr1LBjziwNYZzRXQz',
+                        'Integrity': '1Kyr_bJ5MSN3b6w1KMZzdJ7YPyTgPxze1',
+                        'DTS': '100OsZP0JQIk9_VXHXpK8mdhS9GOp3yKm'}

```

Comment:
Few things about this

* This dictionary should be moved to a config json file (name it what you want). 
* Read in the dictionary from the json file
* Explain how to find and add the folder keys to the file
* Wrap the parentFolder = parentDictionary[parent] in a try except statement to catch key errors (in case the user enters a folder name that isnt present in the dictionary). This will look like

```
try:
     parentFolder = parentDictionary[parent]
except KeyError:
    logging.error("some useful error message here")
    raise KeyError

```  
* Also check that the config file is present - if it is not, raise an error

Reply:
I was thinking of putting parentFolder = parentDictionary[parent] in a while loop that checks if the folder name entered is in the keys of the dictionary. and if not, it would ask again. I believe it would be faster than raising an error and running the script from the beginning again. what do you think about that?

Reply:
Daniel: Instead of raising an error - use a default parent folder